lib/Doctrine/Inflector/Rules/English/Uninflected.php

```diff
@@ -43,147 +43,396 @@ public static function getPlural(): iterable
+        yield new Pattern('botany');
+        yield new Pattern('bravery');
+        yield new Pattern('bread');
         yield new Pattern('bream');
         yield new Pattern('breeches');
         yield new Pattern('britches');
         yield new Pattern('buffalo');
+        yield new Pattern('business');
         yield new Pattern('butter');
+        yield new Pattern('cake');
         yield new Pattern('cantus');
+        yield new Pattern('carbon');
         yield new Pattern('carp');
+        yield new Pattern('cash');
         yield new Pattern('cattle');
+        yield new Pattern('chaos');
         yield new Pattern('chassis');
+        yield new Pattern('cheese');
+        yield new Pattern('chess');
+        yield new Pattern('childhood');
+        yield new Pattern('chocolate');
+        yield new Pattern('clamp');
         yield new Pattern('clippers');
         yield new Pattern('clothing');
         yield new Pattern('coal');
         yield new Pattern('cod');
+        yield new Pattern('coffee');
         yield new Pattern('coitus');
+        yield new Pattern('commerce');
         yield new Pattern('compensation');
+        yield new Pattern('confidence');
+        yield new Pattern('confusion');
         yield new Pattern('congoese');
+        yield new Pattern('content');
         yield new Pattern('contretemps');
+        yield new Pattern('cookery');
         yield new Pattern('coreopsis');
         yield new Pattern('corps');
+        yield new Pattern('corruption');
         yield new Pattern('cotton');
+        yield new Pattern('countryside');
+        yield new Pattern('courage');
+        yield new Pattern('crockery');
+        yield new Pattern('cutlery');
+        yield new Pattern('damage');
+        yield new Pattern('dancing');
+        yield new Pattern('danger');
+        yield new Pattern('darkness');
         yield new Pattern('data');
         yield new Pattern('debris');
         yield new Pattern('deer');
+        yield new Pattern('delight');
+        yield new Pattern('democracy');
+        yield new Pattern('depression');
+        yield new Pattern('designs');
+        yield new Pattern('dessert');
+        yield new Pattern('determination');
         yield new Pattern('diabetes');
+        yield new Pattern('dignity');
+        yield new Pattern('dirt');
+        yield new Pattern('distribution');
         yield new Pattern('djinn');
+        yield new Pattern('driving');
+        yield new Pattern('dust');
+        yield new Pattern('duty');
+        yield new Pattern('earth');
+        yield new Pattern('economics');
         yield new Pattern('education');
         yield new Pattern('eland');
         yield new Pattern('elk');
         yield new Pattern('emoji');
+        yield new Pattern('employment');
+        yield new Pattern('engineering');
+        yield new Pattern('entertainment');
+        yield new Pattern('enthusiasm');
+        yield new Pattern('environment');
```

Comment:
@judaschwartz @greg0ire hi there. This broke Laravel's test suite. `environment` is definitely a countable word: https://en.wiktionary.org/wiki/environment

<img width="522" alt="Screenshot 2023-06-16 at 14 17 06" src="https://github.com/doctrine/inflector/assets/594614/c788346d-15b9-4601-a27a-51c76811aafa">

Can we please revert this and tag a patch release? Right now, all of our PR's are failing. Thanks!

Reply:
I've sent in a PR for this: https://github.com/doctrine/inflector/pull/222